ansible/roles/kubernetes/tasks/setup-Debian.yml

```diff
@@ -24,3 +24,5 @@
       - python-selinux
       - selinux-policy-default
       - snapd
+      - ipvsadm
+      - vim
```

Comment:
Why do we need vim?

Reply:
i use vim inside the cluster, and each time i destroy it and then rebuild i have to reinstall it :) 
so if you don't mind i would like to keep it as basic tool :)

Reply:
/shrug